index.js

```diff
@@ -0,0 +1 @@
+// todo
```

Comment:
This file is suppose to be under `web/`. If you want to put it here, you need to update `entrypoint` in `dfx.json`.